ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -310,4 +318,98 @@ export class IndexerClient {
     };
     return this.queryIndexer(graphqlQuery);
   }
+
+  /**
+   * Queries account's current tokens.
+   * This query queries for any type of tokens (i.e NFT, Fungiable, soul bound, etc).
+   *
+   * @returns GetAccountTokensQuery response type
+   */
+  async getAccountTokens(ownerAddress: MaybeHexString, options?: PaginationArgs): Promise<GetAccountTokensQuery> {
+    const address = HexString.ensure(ownerAddress).hex();
+    IndexerClient.validateAddress(address);
+    const graphqlQuery = {
+      query: GetAccountTokens,
+      variables: { address, offset: options?.offset, limit: options?.limit },
+    };
+    return this.queryIndexer(graphqlQuery);
+  }
+
+  /**
+   * Queries all tokens of a specific collection that an account owns
+   *
+   * @param ownerAddress owner address
+   * @param collectionAddress the collection address
+   * @returns GetAccountCollectionTokensQuery response type
+   */
+  async getAccountCollectionTokensByCollectionAddress(
+    ownerAddress: MaybeHexString,
+    collectionAddress: MaybeHexString,
+    options?: PaginationArgs,
+  ): Promise<GetAccountCollectionTokensQuery> {
+    const address = HexString.ensure(ownerAddress).hex();
+    IndexerClient.validateAddress(address);
+
+    const graphqlQuery = {
+      query: GetAccountCollectionTokens,
+      variables: {
+        collection_id: collectionAddress,
+        owner_address: address,
+        offset: options?.offset,
+        limit: options?.limit,
+      },
+    };
+    return this.queryIndexer(graphqlQuery);
+  }
+
+  /**
+   * Queries a collection data
+   * if, for some reason, a creator account has 2 collections with the same name in v1 and v2,
+   * can pass an optional `tokenStandard` parameter to query a specific standard
```

Comment:
Can we clarify what values are accepted for tokenStandard (e.g. "v1" or "v2" as strings)?

Reply:
`tokenStandard` is of type `TokenStandard` which has only two value options - `v1` and `v2`, it wont let you pass anything else other than that.